src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/utils/NullChecker.java

```diff
@@ -12,6 +15,12 @@ private NullChecker() {
 
     @SuppressWarnings({"java:S3011", "java:S3864"})
     public static boolean anyNonNull(Object target) {
+        if (target == null) {
```

Comment:
Although this code change looks innocuous it's not required and you're changing the function to have a dual purpose. The purpose of the method is to check if any of the fields in the target object are non-null. Now you're also making it check to see if target is a String and has some text. Why? You can use StringUtils.hasText directly in the client code.

Reply:
@al-hmcts @jthmcts 
Updated my change accordingly - https://github.com/hmcts/finrem-case-orchestration-service/pull/1800/commits/10b9a2222d8b2bca7697b4e8d04b2800759108a2